.github/workflows/dev-web.yml

```
@@ -61,20 +60,50 @@ jobs:
             ${{ runner.OS }}-build-${{ env.cache-name }}-
             ${{ runner.OS }}-build-
             ${{ runner.OS }}-
+
       - name: npm install and build
         run: |
           npm ci
           npm run build --if-present
         env:
           CI: "true"
-      - name: Cypress test
+
+      - name: Check build result
+        if: ${{ failure() }}
+        run: echo "Error in build step"; exit 1
+
+      - name: npm start
+        run: |
+          npm start
```

I am not sure if it would be worthwhile defining the port explicitly here? Just where it seems Cypress relies on it being :3000.
```suggestion
          PORT=3000 npm start
```

---

I have added the port to the npm start line.